workflow-templates/git-slice-v2.yml

```diff
@@ -44,12 +44,32 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'workflow_dispatch' }}
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out repository
+        uses: actions/checkout@v2
         with:
           fetch-depth: 1
+
+      - name: Setup Git
+        run: |
+          git config --global user.email ${{ secrets.EMAIL }}
+          git config --global user.name  ${{ secrets.USERNAME }}
+          git config --global pull.rebase true
+          
+      - name: Merge main/master
+        run: |
+          git fetch -f
+          git branch
+          git checkout ${{ github.event.inputs.branch }}
+          git pull origin ${{ github.event.inputs.branch }}
+          git merge origin/master
+          git push origin ${{ github.event.inputs.branch }}
+          git status
+             
       - name: Git Slice Push
         uses: GitStartHQ/actions/GitSlicePush@git-slice
         with:
+          slice_git_token: ${{ secrets.PASSWORD }}
+          slice_git_username: ${{ secrets.USERNAME }}
           upstream_git_email: gitstart@users.noreply.github.com
```

Comment:
@arslanarshad31 Are these being used correctly? the password is the gitslice access token for that bot, and the username would be gitstart-`clientId` for most cases